lib/Parser/Parse.cpp

```diff
@@ -8876,6 +8876,31 @@ ParseNodePtr Parser::ParseCatch()
 
         pnodeCatchScope = StartParseBlock<buildAST>(PnodeBlockType::Regular, isPattern ? ScopeType_CatchParamPattern : ScopeType_Catch);
 
+        if (buildAST)
```

Comment:
ah - you just moved this up. never mind.

---
In reply to: [67058501](https://github.com/Microsoft/ChakraCore/pull/1139#discussion_r67058501) [](ancestors = 67058501)